NotificationApi/NotificationApi.Contract/Requests/AddNotificationRequest.cs

```diff
@@ -1,10 +1,19 @@
+using System;
+using System.Collections.Generic;
+
 namespace NotificationApi.Contract.Requests
 {
     /// <summary>
     /// Add a new notification
     /// </summary>
     public class AddNotificationRequest
     {
-        
+        public string ContactEmail { get; set; }
+        public int NotificationType { get; set; }
+        public string PhoneNumber { get; set; }
+        public Guid HearingId { get; set; }
+        public Guid ParticipantId { get; set; }
+        public int MessageType { get; set; }
+        public Dictionary<string, string> Parameters { get; set; }
```

Comment:
could add summary `///` to this one to explain what they are and where to get a list of

Reply:
Includes data for the hearing such as Case Number, Name, Date, Time and Username